core/src/main/scala/com/broadinstitute/dsp/DbReaderImplicits.scala

```diff
@@ -113,9 +113,18 @@ object DbReaderImplicits {
       case (id, name, cloudContextDb, location, cloudProvider) =>
         cloudProvider match {
           case CloudProvider.Azure =>
-            throw new RuntimeException(
-              s"kubernetesCluster(${id}) is Azure cluster. We should filter out Azure clusters in the query"
-            )
+            AzureCloudContext.fromString(cloudContextDb) match {
+              case Left(value) =>
+                throw new RuntimeException(
+                  s"${value} is not valid azure cloud context"
+                )
+              case Right(value) =>
+                KubernetesCluster(id, name, CloudContext.Azure(value), location)
+            }
+//           TODO: throw exception once we fix query in zombie monitor and resource validator
```

Comment:
Resource Validator does not support Azure yet, but Zombie Monitor does currently handle AKS clusters (see https://github.com/broadinstitute/leonardo-cron-jobs/blob/2fb1c0f194555c36269ed0888ff26762c1abcd14/zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DeletedKubernetesClusterChecker.scala#L40). So I think that the exception is probably inappropriate here.